cloudformation/loadbalancer.yml

```diff
@@ -11,12 +11,29 @@ Outputs:
   ExternalElbHost:
     Description: "Internet-facing load balancer host name"
     Value: !GetAtt ExternalLoadBalancer.DNSName
-  TargetGroup:
-    Description: "Add tasks to this target group"
-    Value: !Ref ExternalTargetGroup
-    # For !ImportValue in application.yml
+  HttpListener:
+    Description: "Add listener rules to this listener"
+    Value: !Ref ExternalListenerHttp
     Export:
-      Name: !Sub "${ClusterName}-TargetGroup"
+      Name: !Sub "${ClusterName}-HttpListener"
+  HttpsListener:
+    Description: "Add listener rules to this listener"
+    Value: !If
+      - HasSslCertificate
+      - !Ref ExternalListenerHttps
+      - "N/A"
+    Export:
+      Name: !Sub "${ClusterName}-HttpsListener"
+  ServiceRole:
+    Description: "ECS Service Role for joining load balancer"
+    Value: !Ref ServiceRole
+    Export:
+      Name: !Sub "${ClusterName}-ServiceRole"
+  VpcId:
```

Comment:
This was the only value from the result of the cluster stack resources lambda function output that `application.yml` needed so I exported it instead of exporting the lambda function ARN.